qemu/tests/smbios_table.py

```diff
@@ -24,6 +24,11 @@ def run(test, params, env):
     notset_output = params.get("notset_output")
     dmidecode_exp = params.get("dmidecode_exp")
     login_timeout = float(params.get("login_timeout", 360))
+    machine_type = params.get("machine_type", "")
+    if ':' in machine_type:
```

Comment:
I actually wrote it that way first, but than I said to myself that if any machine starts using `:` we'll have to fix all the machine-type related code so I'd prefer making this generic as other archs might use it as well. As example currently there is no way to pick type of ppc machine, but we might want to do it and using `:` would allow to share the code... (not that I'd know of anyone working on such feature, it's just that the code was written to be platform-independent as far as noone introduces machine type with `:` in the name)